arm-web/2016-06-01/swagger/connectionGateways.json

```diff
@@ -39,7 +39,10 @@
         "tags": [
           "Gateways"
         ],
-        "operationId": "Gateways_ListBySubscription",
+        "x-ms-examples": {
+          "Task get": { "$ref": "../examples/ConnectionGatewaysGet.json" }
+        },
+        "operationId": "Gateways_ListBySubscriptions",
```

Comment:
I'd recommend removing the 's' at the end of subscriptions, "Gateways_ListBySubscription"